docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+### Would the repair miner have a "budget" that it can spend on delegating new storage miners when previous ones fail? Or are tokens "refunded" when the original storage miner fails, and the repair miner just reallocates it as the budget for a new storage miner contract?
+
+Yes, the repair miner would charge a premium on the underlying storage, assuming some failure rate, and would be responsible for the extra costs if they chose unreliable storage miners to ultimately store the data.
+
+We descoped the repair miner work for mainnet launch as the network is complicated enough without it, and it's relatively straightforward as an upgrade later.
+
+### Does Filecoin reward network or main network line machine need a fixed IP?
+
+For mainnet, you will need a public IP address, but it doesn’t need to be fixed (just accessible).
+
+### What if we lost a sector accidentally, is there any way to fix that?
+
+If you lost the data itself, then no, there’s no way to recover that, and you will be slashed for it. If the data itself is recoverable, though (say you just missed a Windowed PoSt), then the Recovery process will let you regain the sector.
+
+### Has Filecoin confirmed the use of SDR algorithm? Is there any evidence of malicious construction?
+
+SDR is confirmed for mainnet launch, and we have no evidence of malicious construction. The algorithm is also going through both internal and external security audits.
+
+If you have any information about any potential security problem or malicious construction, reach out to our team at security@filecoin.org.
+
+### How likely is the NSE algorithm to come on line later? Is AMD's configuration the most consistent with the current SDR algorithm?
+
+About NSE: NSE is one of the best candidates for a proof upgrade, and teams are working on implementation. But there are other candidates too, which are promising as well. It may be that another algorithm ends up better than NSE -- we don’t know yet. Proof upgrades will arrive after mainnet launch and will coexist.
+
+Yes, AMD may be optimal for SDR, [see this for why](https://github.com/filecoin-project/lotus/blob/master/documentation/en/sealing-procs.md)
+
+### How are you working to bootstrap the demand side of the marketplace? The Discover program is nice but who is the target market for users and how do you get them?
+
+In addition to Filecoin Discover, a number of groups are actively building tools and services to support adoption of the Filecoin network with developers and clients. For example, check out the recordings from our [Virtual Community Meetup](https://filecoin.io/blog/filecoin-virtual-community-meetup-recap/) to see updates about Textile Powergate and Starling Storage.
+
+A number of other programs, like Filecoin Ignite, will also contribute additional demand, tooling, and usage.
+
+### Why not implement the scheme that the white paper says "miners and clients automatically match orders through order books"？Is it because the LIMITS of TPS?
```

Comment:
Does Filecoin have an implementation of client and miner order matching through order books?